src/main/kotlin/uk/gov/dluhc/emsintegrationapi/database/entity/Address.kt

```diff
@@ -0,0 +1,63 @@
+package uk.gov.dluhc.emsintegrationapi.database.entity
+
+import org.hibernate.annotations.CreationTimestamp
+import org.hibernate.annotations.GenericGenerator
+import org.hibernate.annotations.Type
+import uk.gov.dluhc.emsintegrationapi.database.repository.UUIDCharType
+import uk.gov.dluhc.emsintegrationapi.database.repository.UseExistingOrGenerateUUID
+import java.time.Instant
+import java.util.UUID
+import javax.persistence.Entity
+import javax.persistence.EnumType
+import javax.persistence.Enumerated
+import javax.persistence.GeneratedValue
+import javax.persistence.GenerationType
+import javax.persistence.Id
+import javax.persistence.Version
+import javax.validation.constraints.Size
+
+@Entity
+class Address(
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY, generator = "UUID")
+    @GenericGenerator(name = "UUID", strategy = UseExistingOrGenerateUUID.NAME)
+    @Type(type = UUIDCharType)
+    var id: UUID? = null,
+
+    @field:Size(max = 255)
+    val street: String,
+
+    @field:Size(max = 255)
+    var property: String? = null,
+
+    @field:Size(max = 255)
+    var locality: String? = null,
+
+    @field:Size(max = 255)
+    var town: String? = null,
+
+    @field:Size(max = 255)
+    var area: String? = null,
+
+    @field:Size(max = 10)
+    val postcode: String,
+
+    @field:Size(max = 12)
+    var uprn: String? = null,
+
+    @CreationTimestamp
```

Comment:
interested to see if these timestamps are actually being persisted or not without the entity listener declaration.